_posts/2018-03-22-industrialization-versus-data-selection.md

```diff
@@ -0,0 +1,41 @@
+# Brace yourself data selection, industrialization is coming!
+
+Industrialization is one of the most challenging problems for our company and definitely not only ours. In fact, the world of research is far away from productivity. And the question was asked when we thought about building language model (referred as LM in the following) automatically, especially regarding data selection. In fact, our LM was handmade thanks to heuristics built by empirical means ... Meaning that we had to experiment to find the best system with our own eyes. And this is so not compatible with automating.
+
+What is data selection you may ask? However, first thing first.
+
+## ASR: Automatic Speech Recognition Prelude
+
+In ASR, all is very dependant of the language we want to recognize. If we explain it really simply, we often consider building two boxes.
+
+- The first one is called acoustic model. It represents the way of speaking. What sounds can be put together to form a word, a sentence. In fact, we represent the language by a series of phoneme. If we watch in [Wiktionary](https://en.wiktionary.org/wiki/phoneme) it's clearer isn't it? Don't confuse with syllable through. We can take an example. The word 'through'(1 syllable) consists of three sounds, three phonemes: 'th' 'r' 'oo'.
+
+- The second one is called language model which we want to build automatically. It represents the distribution of the words in the given language. Thanks to probabilities, the system is capable of picking the best correspondence between the list of phoneme and the words/sentences.
+
+In order to build these boxes, we need data and the more they are relevant, the better it is! That's why we need data selection.
+
+## How we used to do,
+
+As we discussed above, data selection is a very important step while building a system. Like tons of people, we used the [Moore-Lewis method](http://www.aclweb.org/anthology/P10-2041) which was adapted for bilingual use (like translation) by Axelrod&co called [Domain Adaptation via Pseudo In-Domain Data Selection](https://aclanthology.info/pdf/D/D11/D11-1033.pdf). These are very effective ways to select data using two corpora (in and out domain) and comparing the cross-entropy.
+
+this method need people to choose and validate the model. Moreover, something hits us hard. This article turned out to be eight this year and as we all know, eight years is a long period in Computer Science and in Automatic Speech Recognition. So we asked ourselves: has anyone done some work about data selection since this article? And are there any relevant work ready for a more industrialized turn?
```

Comment:
- "this method need people to choose and validate the model" => not really, but this method is not really scalable for two reasons: 1/ the algorithm can't decide when to stop on its own and 2/ the algorithm has an annoying tendency to promote very short to short sentences.
- "hits" => "hit"

I'm not sure about the part "eight years is long"... It's not so true for many domains actually. You probably should emphasize on something like the fact that we were using this method for 8 years instead. It will introduce the next sentence better IMHO.

- "has anyone done some work about data selection" => "has any new work been done in data selection", just a suggestion
- Last sentence is perfect! 😉